spring-rabbit/src/main/java/org/springframework/amqp/rabbit/connection/RabbitConnectionFactoryBean.java

```diff
@@ -629,4 +652,20 @@ protected SSLContext createSSLContext() throws NoSuchAlgorithmException {
 		return SSLContext.getInstance(this.sslAlgorithm);
 	}
 
+
+	/**
+	 * In case the client doesn't provide the sslProperties location or the truststore is null delegate to Java's default
+	 * trust mechanism
+	 * @throws NoSuchAlgorithmException
+	 * @throws KeyManagementException
+	 * @throws KeyStoreException
+     */
+	private void useDefaultTrustStoreMechanism() throws NoSuchAlgorithmException, KeyManagementException, KeyStoreException {
+		SSLContext c = SSLContext.getInstance(this.sslAlgorithm);
+		TrustManagerFactory factory = TrustManagerFactory.getInstance(TrustManagerFactory.getDefaultAlgorithm());
```

Comment:
Spring is an Open Source. Would be better to use expressive names even for variable in `private` methods.
Just because all this code is visible for everyone 😄 
I mean both these `c` and `factory`.